### ISSUE TITLE
Accessibility improvements for fonts (quickJump) - make all UI Font Sizes increasable

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/QuickFolders#461]
we can configure the icon and menu font sizes but the font size of the search box (quickJump) stays small. Also for users with visual impairments the help text beside the quickJump feature cannot be scaled up:

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/06fa5621-db3f-4e93-9bbc-8e32bbbba61a)

I propose that we use the menu font size also as starting point for the other writing in the QuickFolders toolbox, maybe including the QuickFolders label (?). 

Fonts should be similar to the menu font size:

![image](https://github.com/RealRaven2000/QuickFolders/assets/6314674/5bdbcc87-bd80-4bf1-9fcc-8c1c1ec98df1)

Also the QuickFolders "Preferences tab" should have a font size selector at least when shown in a tab.

**Comment by RealRaven2000** [comment]
Implemented in 6.5.1 - Published 16/04/2024
